src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -771,9 +772,31 @@ private async Task EnsureInitializeAsync()
             try
             {
                 // If already initialized, need not be on the UI thread
-                if (!_initialized)
+                if (_initialized)
```

Comment:
Why is this getting refactored?

Reply:
It's due to problem timing problem I recently noticed. 
Also today @jebriede reported [it](https://teams.microsoft.com/l/message/19:00d019d741b44e54baadd67ef7aeb41d@thread.skype/1606247461775?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ad34ac55-6785-4e7c-82c3-7ba405d30c3c&parentMessageId=1606247461775&teamName=NuGet&channelName=Client&createdTime=1606247461775).
![image](https://user-images.githubusercontent.com/8766776/100144755-1898ee80-2e4c-11eb-9fba-c4e64fe331ab.png)

Reply:
Adding unrelated changes to a PR will:
1. Make the PR more difficult to review. I'm in the mindset of changing sync code to async, and this is throwing me off.
2. Makes the PR slower to merge, because of half the PR is ok to merge, but the other half is not, then we can't merge the good stuff until the rest meets quality standards as well.
2. If the issue this is fixing doesn't have an issue in NuGet/Home with the appropriate labels, it won't be included in our release notes when the new version ships.

Reply:
I agree with the general gist by @zivkan. 

This needs to be considered at it's own merit, and given that it's significantly more difficult to review than the rest of the code, it increases how long getting this merged will take.

Reply:
Ok. Created follow up [issue#10352](https://github.com/NuGet/Home/issues/10352) for this and removed this change.